core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/codesystem/AllCodeSystemSearchRequestBuilder.java

```diff
@@ -43,6 +44,11 @@ public AllCodeSystemSearchRequestBuilder filterByIds(final Collection<String> id
 		return this;
 	}
 	
+	public AllCodeSystemSearchRequestBuilder filterByToolingIds(final Iterable<String> toolingIds) {
+		this.toolingIds = ImmutableSet.copyOf(toolingIds);
```

Comment:
This will fail if `toolingIds` is `null`. It is not necessary to wrap the argument into an ImmutableSet here, feel free to remove, and also apply this to the `setFields`, `filterByIds` methods.

Reply:
Done